mmdeploy/utils/test.py

```diff
@@ -312,7 +313,7 @@ def get_model_outputs(model: nn.Module, func_name: str,
     """
     assert hasattr(model, func_name), f'Got unexpected func name: {func_name}'
     func = getattr(model, func_name)
-    model_outputs = func(**model_inputs)
+    model_outputs = func(**copy.deepcopy(model_inputs))
```

Comment:
Why do we deepcopy the inputs?

Reply:
[solov2 in mmdetection-3.x](https://github.com/open-mmlab/mmdetection/blob/3.x/mmdet/models/dense_heads/solov2_head.py) , line 649: `mlvl_cls_scores[lvl] = cls_scores.permute(0, 2, 3, 1)` will change the original `model_inputs`. So the dimensions will not match during calculation.

Reply:
I see, that's right.
I think this is a potential bug of mmdetection.